### DISCUSSION TITLE
Set a custom layername

### DISCUSSION THREAD

**Discussion by DINH-iy** [openlayers/openlayers#14969] [Q&A]
If you create a layer you need to pass in the layername, this is the actual name of the layer from the source. When i use something custom like ol-ext to have control the name i see for each field is the layername, only thing is i dont want to use that name i want to use a different custom name, example:

I have a layer:
 'Actueel_ortho25', 'https://service.pdok.nl/hwh/luchtfotorgb/wmts/v1_0',
 
 and instead of seeing ' Actueel_ortho' id like to be able to set something custom like 'Ortho laag'
 
 how would i achieve this?

**Comment by ahocevar** [comment]
This is a question for ol-ext rather than OpenLayers, I guess.
